server_src/runners/history.js

```diff
@@ -276,35 +271,24 @@ class HistoryCollection {
                 );
                 //Clearing interval
                 clearInterval(interval);
-                console.log(
-                  "Saving timelapse... woop!",
-                  this.unrenderedFileName
-                );
                 const saveHistory = await History.findById(id);
                 saveHistory.printHistory.timelapse = lapse;
                 saveHistory.markModified("printHistory");
                 await saveHistory.save();
-                HistoryClean.start();
+                await HistoryClean.start();
```

Comment:
I added awaits. Otherwise this function might return before the start does. Is that ok?

Reply:
It's the last function in that chain, it shouldn't matter should it? don't mind either way like.